src/lsg/patterns/element/index.tsx

```diff
@@ -45,69 +45,116 @@ export interface StyledPlaceholder {
 }
 
 const StyledElement = styled.div`
-	cursor: default;
 	position: relative;
+	z-index: 1;
 `;
 
 const StyledElementLabel = styled.div`
 	position: relative;
 	display: flex;
-	padding: 9px ${getSpace(Size.L)}px 9px ${getSpace(Size.XL)}px;
-	border-radius: 3px;
-	cursor: pointer;
+	padding: ${getSpace(Size.XS)}px ${getSpace(Size.L)}px ${getSpace(Size.XS)}px ${getSpace(Size.XXL)}px;
 	align-items: center;
-	color: ${colors.black.toString()};
+	color: ${colors.grey20.toString()};
 	position: relative;
+	font-size: 15px;
+	line-height: 21px;
+	z-index: 1;
+	
+	&::before {
+		content: '';
+		display: block;
+		position: absolute;
+		height: 100%;
+		width: 240px;
+		left: 0;
+		top: 0;
+		margin-left: -240px;
+	}
 
 	&:hover {
-		background: ${colors.grey90.toString()};
+		background ${colors.black.toString('rgb', { alpha: 0.05 })};
+		
+		&::before {
+			background: ${colors.black.toString('rgb', { alpha: 0.05 })};
+		}
 	}
 
 	${(props: StyledElementLabelProps) =>
 		props.active
 			? `
-				color: ${colors.white.toString()};
-				background: ${colors.blue40.toString()};
+				color: ${colors.blue.toString()};
+				background: ${colors.blue80.toString()};
+
+				&::before {
+					background: ${colors.blue80.toString()};
+				}
 
                 &:hover {
-                    background: ${colors.blue40.toString()};
+                    background: ${colors.blue80.toString()};
+
+					&::before {
+						background: ${colors.blue80.toString()};
+					}
                 }
 			`
 			: ''};
 	${(props: StyledElementLabelProps) =>
 		props.highlight
 			? `
 			background: ${colors.grey90.toString()};
+
+			&::before {
+				background: ${colors.grey90.toString()};
+			}
 		`
 			: ''};
 `;
 
 const StyledPlaceholder = styled.div`
```

Comment:
@LKuechler Is it possible to show the Placeholder ONLY when an element is being dragged around?

Reply:
Solved with 4f6f148